.vscode/extensions.json

```
@@ -0,0 +1,5 @@
+{
+    "recommendations": [
+        "ms-vscode-remote.remote-containers"
```

I think this should include the black extension referenced in the configuration:
```suggestion
        "ms-vscode-remote.remote-containers",
        "ms-python.black-formatter"
```

---

Well the `black` extension gets installed in the `devcontainer` as [part of its VSCode config](https://github.com/mozilla-services/tecken/pull/2857/files#diff-24ad71c8613ddcf6fd23818cb3bb477a1fb6d83af4550b0bad43099813088686R18), so that should already be covered.

---

ah, i didn't realize the extensions were sandboxed like that, ignore me